test/Models/multi-head-attention.mlir

```diff
@@ -0,0 +1,261 @@
+    %cst_4 = arith.constant dense<1.280000e+02> : tensor<f32>
+    %cst_5 = arith.constant dense<1.000000e+00> : tensor<32x8x1xf32>
+    %cst_6 = arith.constant dense<-1.000000e+09> : tensor<f32>
+    %cst_7 = arith.constant dense<1.250000e-01> : tensor<f32>
+    %cst_8 = arith.constant dense<0.797884583> : tensor<f32>
+    %cst_9 = arith.constant dense<5.000000e-01> : tensor<f32>
+    %cst_10 = arith.constant dense<3.000000e+00> : tensor<f32>
+    %cst_11 = arith.constant dense<4.471500e-02> : tensor<f32>
+    %cst_12 = arith.constant dense<1.000000e+00> : tensor<f32>
+    %cst_13 = arith.constant dense<9.99999996E-13> : tensor<f32>
+
+    //%transformer_layer_0_self_attention_attention_output_bias = util.global.load @"transformer/layer_0/self_attention/attention_output/bias" : tensor<128xf32>
+    //%transformer_layer_0_self_attention_attention_output_kernel = util.global.load @"transformer/layer_0/self_attention/attention_output/kernel" : tensor<2x64x128xf32>
+    //%transformer_layer_0_self_attention_key_bias = util.global.load @"transformer/layer_0/self_attention/key/bias" : tensor<2x64xf32>
+    //%transformer_layer_0_self_attention_key_kernel = util.global.load @"transformer/layer_0/self_attention/key/kernel" : tensor<128x2x64xf32>
+    //%transformer_layer_0_self_attention_query_bias = util.global.load @"transformer/layer_0/self_attention/query/bias" : tensor<2x64xf32>
+    //%transformer_layer_0_self_attention_query_kernel = util.global.load @"transformer/layer_0/self_attention/query/kernel" : tensor<128x2x64xf32>
+    //%transformer_layer_0_self_attention_value_bias = util.global.load @"transformer/layer_0/self_attention/value/bias" : tensor<2x64xf32>
+    //%transformer_layer_0_self_attention_value_kernel = util.global.load @"transformer/layer_0/self_attention/value/kernel" : tensor<128x2x64xf32>
+    
+    %transformer_layer_0_self_attention_attention_output_bias = arith.constant dense<"0x04A1A03C2E49A03DFA9BB13D185120BEACB29EBDA5D207BEF8871ABE834D5DBC9A376A3C57370DBEE7EC14BCDE5B193E74D4083DDE9589BED54E703B9E0D87BD4523AF3D3552FFBD9F6E3DBEDA3C9CB927EFCD3C96BBA9BC641A523D760BE0BD57CC0E3EEADD9B3C4C893FBDE425AA3D52D533BDA69A99BDF0C6F4BDCB6906BEA278913D3FEFD9BD72F49ABD706A56BEC0A63DBDC8F3FD3D78842A3B15303FBE8F5F303EFD97313DF8DD09BE7E65703EDB0512BCA5FE343DE5F0DE3DDBE6073DE2D96CBD876CB3BCC615A33E9E00053D0E569F3C7549EE3D3C9EDC3C56EF0DBED56E8D3DE73EEB3DD083D83CFCABC83D684FF43DCB837ABD381A91BC0C2923BD5A7A18BE0C8A0C3E5FAF3B3D54D277BDFA79393BB2E2963CE56564BE57D2163E921CD13D752ED0BD9A8FBABC113F0FBED601EE3C29C4893D2B27583E069FA33D16558C3D1342D8BD5018C53DBA1E2CBD15515E3D0D7E11BEF763D4BD82ADDCBD5319573DAADEAABDB0E90D3D574F983D249D98BDD4ED673D7E9DAABDACC460BD1D3B88BCDBB7AB3D0C21D53DBADCED3C9BE6F9BD3B57E73D75E78BBC930A3D3DF64F6D3DADDF5FBDE11AC8BD191B223E36F68E3CEF0C15BD1B21C93DF3A44DBD20F8A53DE667343BAD7FA53CB946503D4AE137BE6EAD473C95B4CA3C08FB7A3CFFA7453EA46EF03D71DFC23C85B0713D2581BBBD80EA1C3D47E3903D049B0ABD"> : tensor<128xf32>
```

Comment:
For this example, we probably shouldn't use pre-trained weights yet (it's too large!), as we care more about ops than results here. We're building tensor initialization support that we can use with this model later to get actual results.

Reply:
Sounds good. I can remove pre-trained weights. I believe tensor initialization won't initialize weight tensors, right? So I will need to initialize them in the model anyway (may be with `const dense<value>`)

Reply:
For now, `const dense<1.0>` splat should be fine. I'm working on a `tpp-run` option that will replace constant splat in the kernel with random values.